src/components/dashboard/FaceVerification/sdk/FaceTecGlobalState.js

```diff
@@ -0,0 +1,34 @@
+import Config from '../../../../config/config'
+import { FaceTecSDK } from './FaceTecSDK'
+
+// Zoom global state object
+// Like in the Zoom's demo app it contains also
+// initialization function with options preconfigured
+const FaceTecGlobalState = {
+  /**
+   * @var {Error|null}
+   */
+  faceTecCriticalError: null,
+  faceTecSDKInitializing: null,
+
+  /**
+   * Convenience method to initialize the FaceTec SDK.
+   */
+  async initialize() {
+    const { faceTecLicenseKey, faceTecLicenseText, faceTecEncryptionKey } = Config
+
+    if (!this.faceTecSDKInitializing) {
+      // if not initializing - calling initialize sdk
+      this.faceTecSDKInitializing = await FaceTecSDK.initialize(
```

Comment:
we need to cache **promise** itself, not its result
this await is wrong, it should be removed, promise lock won't work with it:
a) this.faceTecSDKInitializing will be set **only** if ininialize success 
b) next await statement will fail because this.faceTecSDKInitializing is a boolean true FaceTecSDK.initialize reresolves to, not a promise
c) as this.faceTecSDKInitializing sets only after initialze promise settled, it still possible to run 2+ initialization in parralel

i pushed a hotfix: https://github.com/GoodDollar/GoodDAPP/commit/a8f2b11549d9263fc27c304924da16e814e71a3c